source/common/json/json_loader.cc

```diff
@@ -743,6 +743,16 @@ ObjectSharedPtr Factory::loadFromYamlString(const std::string& yaml) {
     return parseYamlNode(YAML::Load(yaml));
   } catch (YAML::ParserException& e) {
     throw EnvoyException(e.what());
+  } catch (YAML::BadConversion& e) {
+    throw EnvoyException(e.what());
+  } catch (...) {
```

Comment:
I'm against catching "...".  The things that it catches varies by platform, and it shouldn't be neccessary. It looks like everything from the YAML lib inherits from std::exception.  I say we catch that instead.